src/modules/_partials/layouts/DashboardLayout/AppSidebar/index.tsx

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import { Sidebar, DarkThemeToggle } from 'flowbite-react';
+import {
+  HiHome,
+  HiUsers,
+  HiCurrencyDollar,
+  HiClock,
+  HiDocumentReport,
+  HiChartBar,
+  HiCheckCircle,
+} from 'react-icons/hi';
+import { NavLink } from 'react-router-dom';
+
+import logo from '@/assets/images/logos/logo.svg';
+
+const navLinks = [
+  {
+    name: 'Home',
+    href: '/dashboard',
+    icon: HiHome,
+  },
+  {
+    name: 'Users',
+    href: '/dashboard/users',
+    icon: HiUsers,
+  },
+  {
+    name: 'Performance',
+    href: '/dashboard/performance',
+    icon: HiChartBar,
+  },
+  {
+    name: 'Goals',
+    href: '/dashboard/goals',
+    icon: HiCheckCircle,
+  },
+  {
+    name: 'Payroll',
+    href: '/dashboard/payroll',
+    icon: HiCurrencyDollar,
+  },
+  {
+    name: 'Time tracker',
+    href: '/dashboard/time-tracker',
+    icon: HiClock,
+  },
+  {
+    name: 'Reports',
+    href: '/dashboard/reports',
+    icon: HiDocumentReport,
+  },
+];
+
+const AppSidebar = ({ className = 'w-fit' }) => {
+  return (
+    <div className={className}>
+      <Sidebar aria-label="Sidebar with logo branding example">
+        <Sidebar.Logo href="#" img={logo} imgAlt="Flowbite logo">
```

Comment:
The logo image well seen on dark theme, lets put original logo as this one is not clearly seen on light theme.

Reply:
I see